docs/DEPLOYMENT.md

```diff
@@ -1,32 +1,116 @@
 ## Deployment
 This document covers steps on setting up this repository on various cloud hosting providers.
 
-  - [Pre-requisites](#pre-requisites)
   - [AWS](#aws)
 
-### Pre-requisites
-1. Set up your `zappa_settings.json` file by copying `zappa_settings.example.json` file
-    ```sh
-    cp zappa_settings.example.json zappa_settings.json
-    ```
-2. Update environment variables in your `zappa_settings.json` file based on your environment. For all available settings, see our [Zappa Settings guide](ZAPPA-SETTINGS.md).
-
 ### AWS
 
-Plio utilizes [zappa](https://github.com/zappa/Zappa) to deploy to AWS Lambda. Set up Zappa before starting on the deployment as it requires Zappa credentials.
-
-Deploying with Zappa is very simple:
+#### Pre-requisites
+Deploying on AWS requires a basic understanding of the following tools and services:
+1. Docker
+2. GitHub actions & workflows
+3. GitHub environments and secrets
+4. AWS Elastic Container Registry (ECR)
+5. AWS Elastic Container Service (ECS)
+6. AWS Virtual Private Cloud (VPC)
+7. AWS Fargate
+8. AWS Elastic Load Balancer (ELB)
+9. AWS Elastic IPs
+10. AWS Identity and Access Management (IAM)
+11. AWS Relational Database Service (RDS)
 
 #### Staging
-```sh
-# RUN THIS COMMAND ONLY IF STATIC FILES HAVE CHANGED
-zappa manage dev "collectstatic --noinput"
-zappa update dev
-```
+Setting up staging environment on AWS is pretty straightforward.
+1. Login to your AWS console.
+2. Go to VPC. (skip this step if you've already created a VPC when setting up frontend repository)
+   1. Create a new VPC.
+   2. Name it `plio-staging-vpc`.
+   3. In IPv4 CIDR block, enter `10.0.0.0/28`.
+   4. Click on create button. You will see the new VPC under the list of VPCs.
+   5. Check out this [AWS guide for more details on VPC](https://docs.aws.amazon.com/vpc/latest/userguide/working-with-vpcs.html).
```

Comment:
- I had to attach an internet gateway to the VPC - or else can't use this VPC in the Load Balancer step
- I had to attach Subnets to the VPC - or else can't use this VPC in the Load Balancer step

Reply:
I was not able to attach more than one subnet when I use the CIDR as 10.0.0.0/24

Reply:
10.0.0.0/24

Reply:
one would need to understand a bit about VPCs and subnets and how to interpret the 10.0.0.0/16 convention.

Reply:
also, had to add an entry to the route table to make the subnets publicly accessible. Couldn't have done it probably without this: https://cloudacademy.com/course/aws-virtual-private-cloud-subnets-and-routing/demonstration-creating-a-vpc-and-subnets/